kong/plugins/standard-webhooks/internal.lua

```diff
@@ -0,0 +1,78 @@
+local kong = kong
+local hmac = require "resty.openssl.hmac"
+local tonumber = tonumber
+local ngx = ngx
+local type = type
+
+local HEADER_WEBHOOK_ID = "webhook-id"
+local HEADER_WEBHOOK_SIGN = "webhook-signature"
+local HEADER_WEBHOOK_TS = "webhook-timestamp"
+
+local function getHeader(input)
+  if type(input) == "table" then
+    return input[1]
+  else
+    return input
+  end
+end
+
+local function sign(secret, id, ts, payload)
+  local d, err = hmac.new(secret, "sha256")
```

Comment:
use "mac" please, hmac interface is deprecated and doesn't play with fips mode

mac.new(secret, "HMAC", nil, "sha256")

Reply:
Will do